2024/XenSummit/cfp/trenchboot.md

```diff
@@ -0,0 +1,84 @@
+# CFP
+
+## Session title
+
+Challenges and status of enabling TrenchBoot in Xen hypervisor
+
+## Description
+
+The presentation will briefly describe the recent development and progress of
+enabling TrenchBoot in Xen hypervisor, the challenges we had to overcome and
+resulting changes to the Xen hypervisor project. The security properties of a
+system running Xen with TrenchBoot support will be explained. A short demo of
+running DRTM with Xen hypervisor will be shown. Also, the incoming UEFI support
+and the impact on Xen UEFI boot process shall be discussed.
```

Comment:
@ miczyg1, please explain the stack we plan to test. I know you want to keep commitment generic, but if we will fail in given scenario we can say that and explain details during presentation. When describing stack we should mention Dasharo for those platform on which we plan to have it.

Reply:
c6d89a66c724e88f4d07ba89dbc48f4a2dc0933b

Reply:
@miczyg1 much better.